ReactAndroid/src/main/java/com/facebook/react/uimanager/UIImplementation.java

```diff
@@ -43,6 +43,8 @@
  * shadow node hierarchy that is then mapped to a native view hierarchy.
  */
 public class UIImplementation {
+  // Lock needed in order to fix https://github.com/facebook/react-native/issues/17178#issuecomment-395858863
```

Comment:
I like comments in the code, but this particular one is not so helpful. Every single line of the code has long history and motivation, so we cannot/should not express that in comments describing the story behind them.

Comments should describe tricky code in cases where the code itself cannot express this trickiness. Here the name and intent is clear from the variable name and usage.

Reply:
Got it that makes sense, I'll remove the comments

Reply:
Done